### DISCUSSION TITLE
Achieve line breaks within bullet points for primeng editor

### DISCUSSION THREAD

**Discussion by Sakthikrishna13-10** [primefaces/community#201] [PrimeNG] [answered]
In current prime ng editor, i need to achieve line break within bullet points
For example,
This is normal text
        1. this is bullet point text
            here is a line break (with shift+ enter) -

when i click shift + enter after(1. this is bullet point text), it is creating another bullet point like 2.  Instead it should create line break within the bullet points

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

PrimeNG editor depends on Quill editor unfortunately the issue you've mentioned is related to Quill and we cannot fix the malfunction in another library. Maybe you can open an issue on Quill github repo. 
You can see the issue is in Quill from here https://quilljs.com/playground/#autosave
